examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -44,24 +44,6 @@
           "define": "BASIC_CLUSTER",
           "side": "client",
           "enabled": 1,
-          "commands": [
-            {
-              "name": "ResetToFactoryDefaults",
-              "code": 0,
-              "mfgCode": null,
-              "source": "client",
-              "incoming": 1,
-              "outgoing": 1
-            },
-            {
-              "name": "MfgSpecificPing",
-              "code": 0,
-              "mfgCode": "0x1002",
-              "source": "client",
-              "incoming": 1,
-              "outgoing": 1
-            }
-          ],
```

Comment:
Can you rebase ? We should keep the "MfgSpecificPing" command here. That should be easier to use it now that #4192 has landed.